frontend/src/components/EditAdminTeam.js

```diff
@@ -80,17 +80,17 @@ export default function EditAdminTeam({ user }) {
 		direction="column">
 		<Grid item style={{ width: "100%" }}>
 			<Grid container justify="space-between" style={{ width: "100%" }}>
-				<Typography variant="body1" align="left" gutterBottom>
+				<ParagraphTypography variant="body1" align="left" gutterBottom>
```

Comment:
What advantages does ParagraphTypography have for this story? Does this introduce an extra <p> tag for CSS?